src/test/java/amazin/BaseTest.java

```diff
@@ -3,18 +3,19 @@
 import org.junit.*;
 import org.springframework.test.util.AssertionErrors;
 
+import amazin.model.Book;
+
 public class BaseTest {
 
     protected Book book;
 
     protected static long DEFAULT_ID = 6;
     protected static String DEFAULT_NAME = "The Test";
     protected static String DEFAULT_DESCRIPTION = "A book about testing.";
-    protected static int DEFAULT_ISBN = 666;
+    protected static String DEFAULT_ISBN = "6666666666666";
```

Comment:
?

Reply:
6666 default values were used originaly by Moe so I kept it